### PR TITLE
Move specific architectures to Utils::Architectures

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/50432
modified: lib/main_common.pm
modified: lib/version_utils.pm
modified: tests/installation/welcome.pm
modified: tests/console/validate_lvm_encrypt.pm
modified: tests/sles4sap/saptune.pm

verification test runs:
http://f40.suse.de/tests/3371 (s390x)
http://f40.suse.de/tests/3372 (aarch64)
http://f40.suse.de/tests/3370 (x86_64)
http://f40.suse.de/tests/3379 (ppc64le)
http://f40.suse.de/tests/3386 (YaST lvm-full-encrypt@aarch64)
http://f40.suse.de/tests/3387 (sles4sap saptune)
http://openqa.suse.de/tests/2845045 (lvm-full-encrypt@aarch64)
https://openqa.opensuse.org/tests/921396 (TW Gnome)
https://openqa.opensuse.org/tests/921397 (Leap lvm-full-encrypt)

### CODE REVIEW

File: lib/main_common.pm [hunk 1]
```diff
@@ -20,6 +20,7 @@ use suse_container_urls qw(get_suse_container_urls);
 use autotest;
 use utils;
 use wicked::TestContext;
+use Utils::Architectures;
```
Comment (foursixnine) [inline lib/main_common.pm]: This needs to be 
```suggestion
use Utils::Architectures qw(:ARCH);
```

File: lib/main_common.pm [hunk 2]
```diff
@@ -45,6 +46,10 @@ our @EXPORT = qw(
   init_main
   installyaststep_is_applicable
   installzdupstep_is_applicable
+  is_s390x
```
Comment (foursixnine) [inline lib/main_common.pm]: Utils::Architectures already is defining these symbols. main_common doesn't need to export them.

File: lib/main_common.pm (hunk 1 above)
Comment (Zaoliang) [reply lib/main_common.pm]: updated, thanks!

File: lib/main_common.pm (hunk 2 above)
Comment (Zaoliang) [reply lib/main_common.pm]: yes, thanks

File: lib/Utils/Architectures.pm [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package Utils::Architectures;
+use strict;
+use warnings;
+
+use base qw(Exporter);
+use Exporter;
+use testapi qw(:DEFAULT);
```
Comment (foursixnine) [inline lib/Utils/Architectures.pm]: only check_var is used here, there's no need to import the full testapi :) at least for now.

Comment (SergioAtSUSE) [inline lib/Utils/Architectures.pm]:
```suggestion
use testapi 'check_var';
```

Comment (Zaoliang) [reply lib/Utils/Architectures.pm]: yes, changed, thanks

Comment (Zaoliang) [reply lib/Utils/Architectures.pm]: thanks

**Comment by SergioAtSUSE** [comment]
Let's wait for @foursixnine and not merge this on a Friday evening.

File: lib/main_common.pm [hunk 4]
```diff
@@ -1145,7 +1146,7 @@ sub load_consoletests {
     if (is_opensuse || !is_staging && (check_var_array('SCC_ADDONS', 'asmm') || is_sle('15+') && !is_desktop)) {
         loadtest "console/salt";
     }
-    if (check_var('ARCH', 'x86_64')
+    if (is_x86_64
         || check_var('ARCH', 'i686')
```
Comment (foursixnine) [inline lib/main_common.pm]: I wonder if we are still having tests running on i686 or i586, @DimStar77, @lnussel, @coolo  do you guys have a clue? :)

Comment (SergioAtSUSE) [reply lib/main_common.pm]: @foursixnine, for Tumbleweed i586: https://openqa.opensuse.org/tests/overview?distri=microos&distri=opensuse&version=Tumbleweed&build=20190505&groupid=1

Comment (foursixnine) [reply lib/main_common.pm]: Ah :( it's actually there for both

**Comment by SergioAtSUSE** [comment]
@Zaoliang, please resolve merge conflicts.

**Pull request merged**